css-brochure-form/styles.css

```diff
@@ -19,6 +19,7 @@ label {
   font-weight: 600;
   font-size: normal;
   letter-spacing: 1px;
+  display: block;
```

Comment:
`div` elements are `block` by default, this shouldn't be necessary 🤔 . If this is for the `label`, you should do that with a separate rule from the rule to set up the size of the form.